packages/edit-site/src/components/global-styles/screen-colors.js

```diff
@@ -46,11 +39,11 @@ function ScreenColors( { name, variation = '' } ) {
 				) }
 			/>
 
-			<BlockPreviewPanel name={ name } variation={ variationClassName } />
+			<BlockPreviewPanel />
```

Comment:
@youknowriad Is there a reason for rendering these `BlockPreviewPanel` instances that have no props? If the `name` prop is missing, there's no block to preview and the component is guaranteed to render `null`.